tethysext/atcore/mixins/attributes_mixin.py

```diff
@@ -35,7 +35,7 @@ def get_attribute(self, key):
             varies: value of attribute.
         """
         if key not in self.attributes:
-            return None
+            return default
 
         return self.attributes[key]
```

Comment:
self.attributes seems to be a normal dictionary so we could just return self.attributes.get(key, default)

Reply:
Good suggestion. Thanks.